vehicle/simvehicle.cc

```diff
@@ -1510,7 +1510,7 @@ void vehicle_t::calc_image()
 {
 	image_id old_image=get_image();
 	// When loading savedata, vehicles do not have cnv information.
-	const bool is_reversed = !cnv||(get_waytype()==air_wt)?false:cnv->is_reversed();
+	const bool is_reversed = !cnv  ||  cnv==NULL  ||  cnv==(convoi_t *)1  ?false:cnv->is_reversed();
```

Comment:
`!cnv` は、 `cnv==NULL` と同義です。
```suggestion
	const bool is_reversed = (cnv==NULL  ||  cnv==(convoi_t *)1) ? false : cnv->is_reversed();
```